web/elm/src/Build/Styles.elm

```diff
@@ -141,10 +142,16 @@ triggerTooltip =
     ]
 
 
-stepHeader : List (Html.Attribute msg)
-stepHeader =
+stepHeader : StepState -> List (Html.Attribute msg)
+stepHeader state =
     [ style "display" "flex"
     , style "justify-content" "space-between"
+    , case state of
+        StepStateFailed ->
+            style "border" "1px solid #C24433"
+
+        _ ->
+            style "border" "none"
```

Comment:
`none` vs. `1px` will result in content slightly jumping when the border is added - similar to https://github.com/concourse/concourse/issues/3078

Reply:
Good snag! Will fix!